src/main/java/org/concordion/internal/ConcordionBuilder.java

```diff
@@ -414,6 +415,7 @@ public ConcordionBuilder withFixture(Fixture fixture) {
 
     public ConcordionExtender withExampleListener(ExampleListener listener) {
 		exampleCommand.addExampleListener(listener);
+        executeCommand.addExampleListener(listener);
```

Comment:
Because tables are executed with the execute command, we need to give example listners there - in case the table is executing as examples.